components/button/Button.tsx

```diff
@@ -1,21 +1,25 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import * as React from 'react';
```

Comment:
Will change the imports to default, because we use `useSyntheticDefaultImports` flag